ProgramTask10.3.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+class Program
+{
+    static void Main(string[] args)
+    {
+        Console.WriteLine("Введите количество чисел (n): ");
+        int n = int.Parse(Console.ReadLine());
+
+        int[] numbers = new int[n];

```

Comment:
Нет проверки введенного числа. Например, если ввести 0, то получится неверное сообщение

> Введите количество чисел (n):
0
Введите числа:
Максимальное значение числа идущих подряд одинаковых чисел: 1

А если  ввести отрицательное чило, возникает исключение 

> Необработанное исключение: System.OverflowException: Переполнение в результате выполнения арифметической операции.

На самом деле программа не может инициализировать массив с отицательным числом элементов
```suggestion
        int n = int.Parse(Console.ReadLine());

        int[] numbers = new int[n];
        
        if (n < 1)
        {
            Console.WriteLine("число n должно быть натуральным");
            return;
        }

```